src/commcare_cloud/terraform/modules/ga_alb_waf/main.tf

```diff
@@ -51,7 +51,10 @@ resource "aws_wafv2_ip_set" "permanent_block" {
   scope              = "REGIONAL"
   ip_address_version = "IPV4"
   description        = "Manually Added IPs for denial"
-  addresses          = ["195.54.160.21/32", "64.39.99.208/32"]
+  addresses          = []
```

Comment:
Should/could we add a comment here to note that these are managed in the AWS console?